prompts/mcp/postgres.md

```diff
@@ -3,10 +3,8 @@ mcp:
   - container:
       image: mcp/postgres:latest
       workdir: /app
-      secrets:
-        postgres.url: POSTGRES_URL
       command:
-        - $POSTGRES_URL
+        - postgresql://host.docker.internal:5432/mydb:
```

Comment:
We still need to offer some way to configure what instance it connects to.

Reply:
I'll fix that